broker-cli/broker-daemon-client/index.js

```diff
@@ -17,11 +20,19 @@ const PROTO_PATH = path.resolve(__dirname, '..', 'proto', 'broker.proto')
  */
 const DEFAULT_RPC_PORT = 27492
 
+/**
+ * Root path from the current module used to resolve cert file paths
+ * @constant
+ * @type {String}
+ * @default
+ */
+const PROJECT_ROOT_DIR = '../'
```

Comment:
This doesn't feel necessary?

Reply:
I didnt want to have a magic string for the `path.join` below.